templates/3_Pixie_Cluster_Cells.ipynb

```diff
@@ -189,16 +189,7 @@
    "cell_type": "markdown",
```

Comment:
<p>I think it would still be helpful to save the version of this table that is not size normalized (i.e. cluster_counts).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/885/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>